docs/ref/contrib/admin/index.txt

```diff
@@ -2306,8 +2311,7 @@ The ``InlineModelAdmin`` class adds or customizes:
     A list or tuple containing extra CSS classes to apply to the fieldset that
     is rendered for the inlines. Defaults to ``None``. As with classes
     configured in :attr:`~ModelAdmin.fieldsets`, inlines with a ``collapse``
-    class will be initially collapsed and their header will have a small "show"
-    link.
+    class will be initially collapsed using an expandable widget.
```

Comment:
@django/triage-review `InlineModelAdmin.classes` references `ModelAdmin.fieldsets` which has a `versionchanged` note, do you think that same version changed note is also required here for `InlineModelAdmin.classes`?

Also suggestions are welcome around tweaking the release notes/version changed note/docs updates. I've done my best but not quite mastered the art of good docs. ❤️

Reply:
I'm also unsure about this. If one source references the other, then it's probably OK to list a versionchanged with the latter only as a single source of truth. But I am unaware of Django's practices here.

Reply:
I will tweak these docs (if necessary) as part of my review process.

Reply:
I tweaked docs and duplicated the `versionadded` note for `InlineModelAdmin.classes`. Explicit is better than implicit.